src/prompts/index.tsx

```diff
@@ -198,6 +216,61 @@ export class PromptManager {
         return false;
     }
 
+    private maybePersistReminder(prompt: PromptConfig) {
+        const { uiActions } = prompt as PromptUiActions;
+
+        if (!uiActions) {
+            return;
+        }
+
+        const reminder = uiActions.decline.type === 'remind'
+            ? { declinedOn: Date.now() }
+            : 'suppressed';
```

Comment:
Can we make the conditionals cover all cases in case we add new states and have fallthroughs?

Reply:
Updated this now 👍   - fallback and added a warning re fallback, but as above - probs unlikely to see it in normal usage.